tests/contrib/operators/test_mysql_to_gcs_operator.py

```diff
@@ -106,7 +121,7 @@ def _assert_upload(bucket, obj, tmp_filename, mime_type=None):
         op.execute(None)
 
         mysql_hook_mock_class.assert_called_once_with(mysql_conn_id=MYSQL_CONN_ID)
-        mysql_hook_mock.get_conn().cursor().execute.assert_called_once_with(SQL)
+        mysql_hook_mock.get_conn().cursor().execute.assert_called_with(SQL)
```

Comment:
Please test that you executed the `tz_query` before you execute the actual `sql` statement. You can test that via [assert_has_calls](https://docs.python.org/3.6/library/unittest.mock.html#unittest.mock.Mock.assert_has_calls). Also see this https://github.com/apache/airflow/pull/5101#discussion_r275438792